tests/encoding_test.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
 # pylint:disable=protected-access
+from __future__ import unicode_literals
```

Comment:
you've changed the semantics of all the tests (and redundantly added `u'...'` in a bunch of places)